src/snowflake/connector/gcs_util.py

```diff
@@ -44,9 +56,8 @@ def create_client(stage_info,
 
         if security_token:
             logger.debug("len(GCS_ACCESS_TOKEN): %s", len(security_token))
-            logger.debug("GCS operations with an access token are currently "
-                         "unsupported")
-            client = None
+            logger.debug("Access token is saved as client for renew")
+            client = security_token
```

Comment:
I can understand why, but I'm wondering if we have other alternatives, like directly setting the token on meta['access_token'] or something like that..
@sfc-gh-mkeller any thoughts here?

Reply:
I have no preference. I'll let @sfc-gh-mkeller choose the style.